consider ARCH as ppc64le when rust arch is powerpc64

Amulyam24 opened the pull request:
The rust target arch for `ppc64le` is `powerpc64`.  However, while resolving the platform for an image, consider it as `ppc64le`.

flavio commented:
@Amulyam24 can you rebase your PR and address the broken tests? Ignore the ones about `cargo deny`, they are going to be fixed by a PR I just merged (hence the request to rebase)

Amulyam24 commented:
> @Amulyam24 can you rebase your PR and address the broken tests? Ignore the ones about `cargo deny`, they are going to be fixed by a PR I just merged (hence the request to rebase)

Sure @flavio. Rebased and fixed the issue.

Amulyam24 commented:
@flavio, is there an estimated time period for the next release?
